tests/integration/issues/github_3124/test_cli_executor.py

```diff
@@ -27,7 +27,9 @@ def test_executor_cli_docker(docker_image):
 
 
 def test_zed_runtime_cli_docker(docker_image):
-    process = subprocess.Popen(['jina', 'runtime', '--uses', 'docker://clitest:latest'])
+    process = subprocess.Popen(
+        ['jina', 'executor', '--uses', 'docker://clitest:latest']
```

Comment:
but we do not want to fail. Or we expect this not to work? We will need to do changes in the tests in the hub